assets/js/components/developer-list.vue

```diff
@@ -13,9 +13,29 @@
                     </v-list-tile-avatar>
                     <v-list-tile-content>
                         <v-list-tile-title>
-                            <v-progress-linear v-if="!item.done" v-bind:indeterminate="true"></v-progress-linear>
-                            <v-progress-linear v-else :value="item.total * 4"></v-progress-linear>
+                            {{item.name}}
                         </v-list-tile-title>
+
+                        <v-list-tile-sub-title>
```

Comment:
I think the `v-if` condition can be moved to this level since all the child elements needs to be initially hidden